.scripts/run_osx_build.sh

```diff
@@ -6,10 +6,22 @@ echo -e "\n\nInstalling a fresh version of Mambaforge."
 if [[ ${CI} == "travis" ]]; then
   echo -en 'travis_fold:start:install_mambaforge\\r'
 fi
-MAMBAFORGE_URL="https://github.com/conda-forge/miniforge/releases/latest/download"
-MAMBAFORGE_FILE="Mambaforge-MacOSX-x86_64.sh"
-curl -L -O "${MAMBAFORGE_URL}/${MAMBAFORGE_FILE}"
-bash $MAMBAFORGE_FILE -b
+if [[ ! -z ${CONDA_EXE} ]]; then
+    # Already installed, and activated
+    CONDA_BASE_PREFIX=$(conda info --base)
```

Comment:
It may be that we want to source the profile script in here too. if the user doesn't run in interactive mode, it won't be source by default.